typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -34,6 +34,8 @@ import {
 } from './types';
 
 export class HyperlaneIsmFactory extends HyperlaneApp<FactoryFactories> {
+  public deployedIsms: ChainMap<any> = {};
```

Comment:
Can we avoid any here. It looks like it's a ChainMap<ChainMap<ModuleType>>?

Reply:
It's more of a `ChainMap<Address> | ChainMap<ChainMap<Address>>`. @yorhodes was ok with using `any` since this part of the codebase is due to be changed soon anyway iiuc